routes/faucet.ts

```diff
@@ -23,12 +28,34 @@ const counterDripError = new client.Counter({
 router.post(
   "/",
   ensureAuthenticated,
+  decorateGithubUser,
   blockedAddresses,
   rateLimit,
   async (req: any, res: any, next: any) => {
     const { address } = req.body;
 
     try {
+      if (req.user.github) {
+        const oneQuarter = 7.776e9;
+        const dateSince =
+          new Date().getTime() - new Date(req.user.github.created_at).getTime();
+
+        if (dateSince < oneQuarter) {
```

Comment:
can we use (configurable) days?

Reply:
good call

Reply:
resolved, provide days via env, default 90.